server/src/DBTablesAction.cc

```diff
@@ -1114,7 +1115,7 @@ string ActionsQueryOption::Impl::makeConditionTemplate(void)
 	// host_id;
 	const ColumnDef &colDefHostId = COLUMN_DEF_ACTIONS[IDX_ACTIONS_HOST_ID];
 	cond += StringUtils::sprintf(
-	  "((%s IS NULL) OR (%s=%%" PRIu64 "))",
+	  "((%s IS NULL) OR (%s=%%s))",
```

Comment:
I mean we should use `"%" FMT_LOCAL_HOST_ID` instead of `"%s"`.
(`"%" FMT_LOCAL_HOST_ID` is equivalent with `"%s"` but more meaningful.)
And the around code should use FMT_* macros too.